semantic-conventions/README.md

```diff
@@ -2,6 +2,23 @@
 
 A docker image to process Semantic Convention YAML models.
 
+## Development
+
+**Build Dependencies**:
+
+- pip
+- docker
+
+**Build**:
+Run the scripts described in `.github/workflows/semconvgen.yml`
+
+Additionally, the following python modules need to be on the path
+
+```
+pip install -r dev-dependencies.txt
+pip install -r ./src/semconvgen.egg-info/requires.txt
```

Comment:
This should not be needed. First time I have seen this. Also, this directory would only be generated as a byproduct of `pip install -U -e .` (and the `-e` should really be changed to `--editable` in the yml to be more clear), which is what you should do instead to install the runtime dependencies.

Reply:
Python is not my "day-to-day" programming language, so every time I have to do something with it, I have to remember the steps, so it would be actually great to have a getting started guide with all the steps for newbies like me :D